source/cloud/nvidia/brev.md

```diff
@@ -74,34 +99,41 @@ ssh <username>@<ip> -p 2222
 
 Replace `username` with your username and `ip` with the ip listed if it's different.
 
-### Accessing the Service
+##### Accessing the Service
 
 After running the command, open your web browser and navigate to your local host. You will be able to access the Jupyter Notebook running in your Brev environment as if it were running locally.
 
-### Access the Jupyter Notebook via the Tunnel
+#### 3. Access the Jupyter Notebook via the Tunnel
 
 The "Deployments" section will show that your Jupyter Notebook is running on port `8888`, and it is accessible via a shareable URL Ex: `jupyter0-i55ymhsr8.brevlab.com`.
 
 Click on the link or copy and paste the URL into your web browser's address bar to access the Jupyter Notebook interface directly.
 
-#### Share the Service
+##### 4. Share the Service
 
 If you want to share access to this service with others, you can click on the "Share a Service" button.
 
 You can also manage access by clicking "Edit Access" to control who has the ability to use this service.
 
-## Check that your notebook has GPU Capabilities
+### Check that your notebook has GPU Capabilities
 
 You can verify that you have your requested GPU by running the `nvidia-smi` command.
 
 ![Screenshot of a notebook terminal running the command nvidia-smi and showing the NVIDIA T4 GPU in the output](/_static/images/platforms/brev/brev6.png)
 
-## Test RAPIDS
+### Test RAPIDS
 
 You can verify your RAPIDS installation is working by importing `cudf` and creating a GPU dataframe.
 
 ![Screenshot of a notebook cell importing and using cudf](/_static/images/platforms/brev/brev7.png)
 
-### Resources
+
+
+### Resources and tips
 
 - [Brev Docs](https://brev.dev/)
+- Please note: Git is not preinstalled in the RAPIDS container, but can be installed into the container when it is running using
+```
+apt update
+apt install git -y
+```
```

Comment:
This feels like pretty broad advice. Do we want to mention this in more places? Or should we consider adding `git` to the container if the requirement is strong enough that we need to document it?